index.bs

```diff
@@ -402,54 +379,10 @@ Angular acceleration of the sensor given in radians per second squared. MAY be
 null if the sensor is incapable of providing angular acceleration. When not null
 MUST be a three-element array.
 
-<div class="example">
-The following code snippet creates a WebGL-compatible matrix from a
-{{VRPose}}:
+<dfn attribute for="VRFrameData">poseMatrix</dfn>
+A 4x4 matrix describing the tranform defined by the position and orientation of
+the pose, given as a 16 element array in column major order.
 
```

Comment:
At the very least we need some way to indicate that tracking was lost for 6dof supporting devices whose fallback "3dof" mode still contains neck modeling.